packages/proxy/test/unit/http/request-middleware.spec.ts

```diff
@@ -83,7 +83,7 @@ describe('http/request-middleware', () => {
 
         expect(ctx.req.headers!['x-cypress-is-from-extra-target']).not.to.exist
         expect(ctx.req.isFromExtraTarget).to.be.true
-        expect(ctx.onlyRunMiddleware).to.be.calledWith(['MaybeSetBasicAuthHeaders', 'SendRequestOutgoing'])
+        expect(ctx['onlyRunMiddleware']).to.be.calledWith(['MaybeSetBasicAuthHeaders', 'SendRequestOutgoing'])
```

Comment:
I originally fixed this due to cache issues on the mac machines, but then also realized that the `stop-only` script fails in local development in spots where it shouldn't. So I think these changes are warranted as the `stop-only` shouldn't be throwing for these as they are `.only` mocha tests.